### ISSUE TITLE
Sprint 60 priorities - Facilities

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#9317]
<details><summary>Background</summary>

🥅   — Sprint goals (pending Sprint planning)
💡 — Design
💧  — Drupal engineering
🎉 — FE
🎓  — Training / Change management / KB
📟 — Helpdesk
🔮  —  Planning / roadmap / epic refinement
📊  —  Analytics
🚰  — Service design
🌱 — Good first task

</details>

## #9096 Facility COVID status post-launch

Change management
- [ ] #9195 

## Facility Locator

- [x] #9320 
- [x] #9321
- [x] #9067 🥅 
- [x] #8870  
- [ ] #8769 (David, possibly with a designer)
- [x] #9324

Refine
- [ ] Bounding box to Radius got rolled back to bounding box, possibly low priority to go back to radius.

</details>

## VAMC

- [ ] #9220 (FE prod defect, Billing and insurance)

###  #3151 3 VAMC Top task pages

Support for ongoing change management.

### #7622 Lovell

Background: See epic https://github.com/department-of-veterans-affairs/va.gov-team/issues/34022

#### Research

- [x] #9070 🥅 

<details><summary>CMS (on pause until design is validated)</summary>

- [x] #7626 (sprint 55)
- [ ] Create proof of concept
- [ ] CMS design discovery epic
  - [ ] #8863
- [ ] Refine epic #1939 (for DoD facilities)
  - [ ] #1940 
  - [ ] #1941 
  - [ ] #1942 
  - [ ] #1943  

</details> 

##  #9318 Hours

[Mural for reference](https://app.mural.co/t/vagov6717/m/vagov6717/1653327582500/26b235b5a21b63f6f30cf209d51303f53062c0d4?sender=kevinwalsh8610)

- [x] #9326 
- [x] #9323  

## Events V2 and Recurring events

Some capacity for Ethan/Max for followup 

## Lighthouse / Facility API data

- [x] #7279 (Jay) 🥅

<details><summary>Blocked by V1</summary>

- [ ] #8017  
- [ ] #4769 

</details>

## For awareness from other teams

- #9097 (Sitewide CMS)

## Refinement

Thursday
- [ ] #7490
- [ ] #9390 (FE)
- [ ] #8567 (David, Jay)
- [ ] FL: Bounding box to Radius got rolled back to bounding box, possibly go back to radius. [See issues here](https://github.com/department-of-veterans-affairs/va.gov-team/issues?q=is%3Aissue+is%3Aopen+bounding+box)

Not Thursday
- [x] #9323 
- [ ] #7707 and #9286 
- [ ] Geodata for Vet Center CAPs 
- [ ] Q3 planning
